Framework/Kernel/inc/MantidKernel/CompositeValidator.h

```diff
@@ -7,6 +7,7 @@
 #include <boost/make_shared.hpp>
 
 #include <list>
+#include <sstream>
```

Comment:
Is this header required? This gets included in a lot of other places so it's good to keep these core headers as small as possible.

Reply:
No. Will move it.

Reply:
:+1: